config/main.go

```diff
@@ -46,6 +47,13 @@ type CliConfig struct {
 	QueryPath  string `json:"query_url"`
 }
 
+type DiscordPlatformConfig struct {
+	ApplicationID        string `json:"application_id"`
+	PublicKey            string `json:"public_key"`
+	BotToken             string `json:"bot_token"`
+	ProofServerChannelID string `json:"proof_server_channel_id"`
+}
```

Comment:
`DiscordPlatformConfig` struct doesn't match `config/config.sample.json` modification. 

Which one is correct?